test/acceptance/features/bindAppToMultipleServices.feature

```diff
@@ -108,11 +108,11 @@ Feature: Bind a single application to multiple services
             apiVersion: binding.operators.coreos.com/v1alpha1
             kind: ServiceBinding
             metadata:
-                name: binding-request-1sbr
+                name: $scenario_id-binding
```

Comment:
you can also just use `$scenario_id`

Reply:
I'm aware, and an earlier draft of this work had it that way.  I decided to change it to its current form because I found that if I named everything `$scenario_id`, it was difficult to keep track of what particular resource was getting referred to in some tests at a glance.  As a specific example, there were a few tests in `bindAppToServiceUsingSecret.feature' where it was hard to tell at a glance if `$scenario_id` was referring to an appconfig, a service binding, or a secret.

For the sake of consistency, I decided to name every binding `$scenario_id-binding` or similar for ease of readability, and I tried to do the same with the other resources that get created in our tests.